docs/www/docs.html

```diff
@@ -58,26 +58,27 @@
     <h3>Translations</h3>
     <ul class="list-unstyled">
         <li>
-            <a href="http://perldoc.jp">perldoc.jp</a>
+            <a href="http://perldoc.jp/">Japanese</a>
         </li>
         <li>
-            <a href="http://www.perl.it/documenti/index.html">Italian</a> (Also <a href="http://pod2it.sourceforge.net/">pod2it</a>)
+            <a href="https://metacpan.org/release/POD2-ES">Spanish</a>
         </li>
         <li>
-            <a href="http://perl.enstimac.fr/DocFr.html">French</a>
+            <a href="https://metacpan.org/release/POD2-RU">Russian</a>
         </li>
         <li>
-            <a href="https://metacpan.org/release/POD2::PT_BR">Brazilian portuguese</a>
+            <a href="https://metacpan.org/release/POD2-PT_BR">Brazilian portuguese</a>
         </li>
         <li>
-            <a href="https://metacpan.org/release/POD2::LT">Lithuanian</a>
+            <a href="http://www.perl.it//documenti/pod2it/">Italian</a>
         </li>
         <li>
-            <a href="https://metacpan.org/release/POD2-ES">Spanish</a>
+            <a href="http://perl.enstimac.fr/DocFr.html">French</a>
         </li>
         <li>
-            <a href="https://metacpan.org/release/POD2-RU">Russian</a>
-</li>    </ul>
+            <a href="https://metacpan.org/release/POD2-LT">Lithuanian</a>
```

Comment:
Is this worth including here?  It has only a tiny amount of coverage. https://metacpan.org/source/KUDARASP/POD2-LT-0.02/lib/POD2/LT